contracts/metatx/MinimalForwarder.sol

```diff
@@ -26,29 +21,64 @@ contract MinimalForwarder is EIP712 {
         bytes data;
     }
 
-    bytes32 private constant _TYPEHASH =
+    struct ExpiringForwardRequest {
+        uint48 deadline;
+        address from;
+        address to;
+        uint256 value;
+        uint256 gas;
+        uint256 nonce;
+        bytes data;
+    }
+
+    bytes32 private constant _FORWARD_REQUEST_TYPEHASH =
         keccak256("ForwardRequest(address from,address to,uint256 value,uint256 gas,uint256 nonce,bytes data)");
+    bytes32 private constant _EXPIRING_FORWARD_REQUEST_TYPEHASH =
+        keccak256(
+            "ExpiringForwardRequest(uint48 deadline,address from,address to,uint256 value,uint256 gas,uint256 nonce,bytes data)"
+        );
 
     mapping(address => uint256) private _nonces;
 
-    constructor() EIP712("MinimalForwarder", "0.0.1") {}
+    constructor() EIP712(name(), version()) {}
```

Comment:
why change from `constructor() EIP712("MinimalForwarder", "0.0.1") {}` ?

Reply:
Replied in your comments about `version()` and `name()`.

Reply:
Same as https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4065#discussion_r1115184979